runtime/description.js

```diff
@@ -225,16 +225,20 @@ class ValueToken {
     }
     if (view.type.isView) {
       let viewList = view.toList();
-      if (viewList) {
-        if (viewList.length > 2) {
-          // TODO: configurable view display format.
-          return `<b>${viewList[0].rawData.name}</b> plus <b>${viewList.length-1}</b> other items`;
+      if (viewList && viewList.length > 0) {
+        if (viewList[0].rawData.name) {
```

Comment:
Why `rawData.name` and not just `name`?

Reply:
entity has 2 data members - id and rawName, but rawName contains all the properties of the schema type.

Reply:
Ok. I think I have PEC confusion. Is true to say that inner-PEC Entity instances have schema-aware getters/setters (and also rawData is a Proxy)?

Reply:
right, on the outer PEC side the entity's rawData is just JSON.

Reply:
Thank you!